docker/docker-compose.yml

```diff
@@ -109,7 +111,8 @@ services:
       - ./users/emailer/templates/${MF_EMAIL_TEMPLATE}:/${MF_EMAIL_TEMPLATE}
     depends_on:
       - users-db
-      - authn
+    # expose:
```

Comment:
We should not really commit commented code like this on master.

Reply:
I've reverted to the master version of `docker-compose.yml`